src/components/table/table.scss

```diff
@@ -39,3 +48,30 @@
   opacity: 1;
   transition: opacity 200ms 50ms ease-in;
 }
+
+.carbon-table__configure-link {
+  @include calc(height, "#{$tables__header-height} - 1px");
+  left: -25px;
+  position: absolute;
+  top: -1px;
+  width: 25px;
+
+  .carbon-link__anchor {
+    background-color: $grey-dark-blue-80;
+    border: 1px solid $border-color;
+    border-right: none;
+    border-radius: 25px 0 0 25px;
+    color: $white;
+    display: flex;
```

Comment:
This is working with `margin: auto;` but I can pre-empt a weird IE11 + WIN10 bug - use:

```
align-items: center;
justify-content: center;
```

in the same selector as `display: flex;` and remove the margin definition from the child.